app/views/jobseekers/confirmations/new.html.haml

```diff
@@ -2,7 +2,7 @@
 
 .govuk-grid-row
   .govuk-grid-column-two-thirds
-    %h1.govuk-heading-l= t(".title")
+    %h1.govuk-heading-xl= t(".title")
```

Comment:
I think we should start with the habit of small PRs. This should be another small PR that does not distract the reviewer from what is the main purpose of the PR

Reply:
I have moved it to a separate PR.